### PR TITLE
Icons navigation, Logos links

### PR DESCRIPTION
**Pull request by MariaSolovyeva**
- removed Browse Map button
- added active state to the button onclick
- show search field on the search button click

### CODE REVIEW

**Comment by MariaSolovyeva** [comment]
screen-493

File: django_project/core/base_static/js/custom-jquery.js [hunk 1]
```diff
@@ -35,7 +35,17 @@ $(document).ready(function () {
             }
         }
     });
-
+	
+	$('.icons-nav a').click(function () {
```
Comment (meomancer) [inline django_project/core/base_static/js/custom-jquery.js]: It is class definition

File: django_project/frontend/templates/map.html [hunk 2]
```diff
@@ -155,8 +155,8 @@
                 <div style="clear: both;"></div>
             </div>
             <div id="icons-nav">
```
Comment (meomancer) [inline django_project/frontend/templates/map.html]: in html, it is in "id" definition

**Comment by meomancer** [comment]
@MariaSolovyeva could you check my comment in previous?
and also, when i change in javascript in #icon-nav, it cannot closed again

File: django_project/core/base_static/js/custom-jquery.js [hunk 3]
```diff
@@ -35,7 +35,17 @@ $(document).ready(function () {
             }
         }
     });
-
+	
+	$('#icons-nav a').click(function () {
```
Comment (meomancer) [inline django_project/core/base_static/js/custom-jquery.js]: there is something wrong with this logic
you could try to change like this

$('#icons-nav a').click(function () {
        if ($(this).hasClass("active")) {
            $(this).removeClass("active");
            $(".search-block").hide();
        }
        else {
            $(this).addClass("active");
            $(".search-block").show();
        }
    });

**Comment by meomancer** [comment]
LGMT

**Pull request merged**